docs/usage/getting-started.md

```diff
@@ -146,3 +146,9 @@ func obtainJWTTokenFromExternalSources() -> DelegatedAuthClosure {
 ```
 
 __Step 5__: Run the sample app
+
+License
+-------
+
+Any use of this software is governed by the attached [Box SDK Beta Agreement](../../BETA-AGREEMENT.md).  If you do not
+accept the terms of the Box SDK Beta Agreement, you may not use this software.
```

Comment:
```suggestion
you accept the terms of the Box SDK Beta Agreement.
```

Reply:
After reviewing "Proposed Changes to GitHub page of Box iOS SDK by Legal.docx" ( https://cloud.app.box.com/s/0fe26b12k1ekkrzpimrnnnn19d89x320) ... never mind.